### DISCUSSION TITLE
Add multiple relative workspaces

### DISCUSSION THREAD

**Discussion by Andriiklymiuk** [thunderclient/thunder-client-support#913] [Ideas]
Hi, 

i really love thunder-client, it is making dev process much more enjoyable.
Thanks for the work you've done!

One feature will be nice to have: multiple workspace relative path of thunder-tests.

### Current functionality
```
mainProjectFolder
   settings.json
   microserviceA
      thunder-tests
   microserviceB
      thunder-tests
   microserviceC
      thunder-tests
```
Currently, if you want to import collections from `microserviceA` to `mainProjectFolder`, you can add in `settings.json` workspace relative path and the collections from `microserviceA` will be shown in the main workspace thunder-client tab.

```
{
  "thunder-client.workspaceRelativePath": "./microserviceA",
}

```
### Problem
This is working fine, but if you want to import thunder-client from `microserviceB` and `microserviceC` there is no automatic way (at least from what I found) to import collections from multiple sub-folders. You can import them manually, after exporting each time from each sub-folder, but it is less convenient and time consuming on every change.

### Suggestion
You can add smth like `.workspaceRelativeMultiplePath` to `settings.json`, so that thunder-client can support multiple collections/environments :
```
{
  "thunder-client.workspaceRelativeMultiplePath": [
    "microserviceA",
    "microserviceB",
    "microserviceC"
  ]
}

```

In this way, the following project structure will be handled better and it will even better improve thunder-client experience.
```
mainProjectFolder
   settings.json
   microserviceA
      thunder-tests
   microserviceB
      thunder-tests
   microserviceC
      thunder-tests
```

Will be happy to help, if needed!

**Comment by rangav** [comment]
Thanks @Andriiklymiuk for the feedback, will review it. This can be very complicated use case

is it not possible to have all the tests in `mainprojectFolder` and link to that from child micro service projects using `workspaceRelativePath?`

**Reply by rangav** [reply]
Another user having multiple projects, he is using workspaceRelativePath setting 
https://github.com/rangav/thunder-client-support/issues/105

**Reply by Andriiklymiuk** [reply]
The issue here is that each microservice doesn't know about `mainprojectFolder` and is developed in separate repos, but `mainprojectFolder` is folder/workspace with all cloned microservices. It helps to run everything in one folder to understand if everything is working correctly

**Reply by Andriiklymiuk** [reply]
> Another user having multiple projects, he is using workspaceRelativePath setting
> #105

hmm, maybe separate folder repo with all thunder-tests will help me, but still will not be ideal.
but thanks anyway)

**Comment by rangav** [comment]
separate project for tests can also work.

You can use the `CustomLocation` setting to provide the full path of the project
